diff -ruN ansible-1.4.4/lib/ansible/constants.py ansible-1.4.4-patched/lib/ansible/constants.py
--- lib/ansible/constants.py	2014-01-08 17:22:36.000000000 +0000
+++ lib/ansible/constants.py	2014-01-30 20:51:16.320015352 +0000
@@ -84,11 +84,7 @@
 
 # Needed so the RPM can call setup.py and have modules land in the
 # correct location. See #1277 for discussion
-if getattr(sys, "real_prefix", None):
-    # in a virtualenv
-    DIST_MODULE_PATH = os.path.join(sys.prefix, 'share/ansible/')
-else:
-    DIST_MODULE_PATH = '/usr/share/ansible/'
+DIST_MODULE_PATH = os.path.join(sys.prefix, 'share/ansible/')
 
 # check all of these extensions when looking for yaml files for things like
 # group variables
